src/components/cuda_pw/api_cuda.h

```diff
@@ -0,0 +1,37 @@
+/*
+    Include appropriate CUDA APIs. Set flags to check which API.
+*/
+#ifndef __SELECT_API_H__
+#define __SELECT_API_H__
+
+    #include <cuda.h>
```

Comment:
I would not indent preprocessor directives for consistency with the rest of the code base.

Reply:
For consistency with code base is ok, but there is considerable nesting in these macros that indentation would improve readability for developer.